spec/models/manageiq/providers/embedded_ansible/provider_spec.rb

```diff
@@ -2,4 +2,44 @@
 
 describe ManageIQ::Providers::EmbeddedAnsible::Provider do
   it_behaves_like 'ansible provider'
+
+  subject { FactoryGirl.build(:provider_embedded_ansible) }
+
+  context "DefaultAnsibleObjects concern" do
+    before do
+      subject.save
```

Comment:
Why not `FactoryGirl.create` rather than adding the `subject.save` in the `before` block?